src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/InitialDataTci.cpp

```diff
@@ -26,6 +27,11 @@ bool DgInitialDataTci::apply(
         subcell_vars,
     double rdmp_delta0, double rdmp_epsilon, double persson_exponent,
     const Mesh<3>& dg_mesh, const TciOptions& tci_options) noexcept {
+  const Scalar<DataVector> tilde_b_magnitude =
+      tci_options.magnetic_field_cutoff.has_value()
+          ? magnitude(get<ValenciaDivClean::Tags::TildeB<>>(dg_vars))
+          : Scalar<DataVector>{};
```

Comment:
How about `... = tci_options.tci_options.magnetic_field_cutoff.value_or(Scalar<DataVector>{});`?

Reply:
but the value is a double, not `Scalar<DataVector>`